vendor/github.com/IBM/secret-utils-lib/pkg/utils/constants.go

```diff
@@ -47,6 +47,8 @@ const (
 	ProdIAMURL = "https://private.iam.cloud.ibm.com"
 	// PublicIAMURL ...
 	PublicIAMURL = "https://iam.cloud.ibm.com"
+	// StagePublicIAMURL ...
+	StagePublicIAMURL = "https://iam.test.cloud.ibm.com"
```

Comment:
variable naming convention does not looks consistent, can you make it

Reply:
As these are vendor file changes , code must be fixed from the library side. Internal issue is opened to address the same.